Tangram.podspec

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Tangram-es",
+  "version": "0.1.0",
+  "summary": "Open source C++ library for rendering 2D and 3D maps from vector data using OpenGL ES.",
+  "description": "Open source C++ library for rendering 2D and 3D maps from vector data using OpenGL ES, wrapped with native Cocoa Touch APIs.",
+  "homepage": "https://mapzen.com/products/tangram/",
+  "license": {
+    "type": "MIT",
+    "file": "LICENSE.md"
+  },
+  "documentation_url": "https://mapzen.com/documentation/tangram/",
```

Comment:
We should start thinking about adding an "iOS API" section to [https://mapzen.com/documentation/tangram/](https://mapzen.com/documentation/tangram/).